src/main/webapp/app/shared/form/title-channel-name/title-channel-name.component.ts

```diff
@@ -24,6 +25,14 @@ export class TitleChannelNameComponent implements OnInit {
         if (!this.channelNamePrefix) {
             this.channelNamePrefix = '';
         }
+
+        if (this.initChannelName) {
+            setTimeout(() => {
```

Comment:
Why is this in a timeout?

Reply:
The child component updates a variable in the parent component. To avoid the "NG0100: Expression has changed after it was checked" error, I have to either put this into a timeout, or call detectChanges() in each parent component manually. Hence, I prefer that the child component handles this by itself, so that a developer cannot forget to call detectChanges() if this component is used somewhere else sometime in the future.